_projects/2024/100446517/100446517.Rmd

```diff
@@ -0,0 +1,692 @@
+                                         "LU" = "LUXEMBOURG",
+                                         "DK" = "DENMARK",
+                                         "NO" = "NORWAY",
+                                         "SE" = "SWEDEN",
+                                         "FI" = "FINLAND",
+                                         "EE" = "ESTONIA",
+                                         "NL" = "NETHERLANDS",
+                                         "IE" = "IRELAND",
+                                         "CZ" = "CZECHIA",
+                                         "AT" = "AUSTRIA",
+                                         "FR" = "FRANCE",
+                                         "SI" = "SLOVENIA",
+                                         "BE" = "BELGIUM",
+                                         "ES" = "SPAIN",
+                                         "PL" = "POLAND",
+                                         "HU" = "HUNGARY",
+                                         "LT" = "LITHUANIA",
+                                         "MT" = "MALTA",
+                                         "PT" = "PORTUGAL",
+                                         "LV" = "LATVIA",
+                                         "SK" = "SLOVAKIA",
+                                         "HR" = "CROATIA",
+                                         "EL" = "GREECE",
+                                         "IT" = "ITALY",
+                                         "RS" = "SERBIA",
+                                         "BG" = "BULGARIA",
+                                         "ME" = "MONTENEGRO",
+                                         "TR" = "TURKEY",
+                                         "RO" = "ROMANIA",
+                                         "CY" = "CYPRUS",
+                                         "DE" = "GERMANY")  # Ensure the dummy space remains unchanged
+
+# Re-plot the graph with full country names
+ggplot(data_aggregated_with_space, aes(x = geo, y = OBS_VALUE, fill = n_book)) +
+  geom_bar(stat = "identity", position = "stack") +
+  scale_fill_manual(values = colors, na.translate = FALSE) + # Ensure no fill for dummy
+  labs(title = "Book reading habits over past 12 months, 2022",
+       subtitle = "(% of people aged 16 and over)",
+       x = NULL,
+       y = "Percentage",
+       fill = " ") + # Remove legend title by leaving it blank
+  scale_y_continuous(breaks = seq(0, 100, by = 10)) +  # Set y-axis ticks at intervals of 10
+  theme_minimal(base_family = "roboto_condensed") +
+  theme(
+    axis.text.x = element_text(angle = 45, hjust = 1),
+    panel.grid.major.x = element_blank(),
+    legend.position = "top",
+    plot.subtitle = element_text(face = "italic"),
+    panel.background = element_rect(fill = "#f5f5f5", color = NA),  # Set panel background
+    plot.background = element_rect(fill = "#f5f5f5", color = NA)    # Set plot background
+  )
+```
+
+### Further adjustments
+
+#### Bold Title
+
+```{r}
```

Comment:
Please add the option `preview=TRUE` so that this image is shown in the gallery.